src/migrations/20250424000000-mark-source-deleted-records.js

```diff
@@ -38,17 +38,18 @@ module.exports = {
         SELECT 1/(LEAST(SUM(beforecount),1) - 1 ) FROM beforecounts;
 
         -- The actual marking of the records as deleted
-        UPDATE "MonitoringClassSummaries" SET "deletedAt" = NOW() WHERE "sourceDeletedAt" IS NOT NULL;
```

Comment:
Why are we updating a migration as part of this change? Because this is in the past it has already been run on prod. Do we need a new migration? Let me know if I am mis understanding this change.

Reply:
Do we need to roll this back?